modin/config/envvars.py

```diff
@@ -404,6 +404,18 @@ def get(self):
         return result
 
 
+class MinElementsInPartition(EnvironmentVariable, type=int):
```

Comment:
I am still thinking this name might be confusing because it may be understood as minimal amount of all elements held in a partition. I would suggest to use `MinPartitionBlockSize`, `MinPartitionSize` or just `PartitionSize`.

Reply:
I am more tend to the latter.

Reply:
I'm against dropping `Min` here, as it implies that partitions would be of exact this size which is wrong. `MinPartitionSize` is fine by me, though.

Reply:
I agree with @YarShev, `PartitionSize` will be clearer definition.

Reply:
`MinPartitionSize` is the most clear to me 😄